src/BotManager.php

```diff
@@ -236,10 +236,10 @@ public function validateAndSetWebhook(): self
                 }
                 return !empty($v);
             }, ARRAY_FILTER_USE_BOTH);
-
+            $webhook['url'] .= ( isset(parse_url($webhook['url'])['query'])) ? '&' : '?' ;
```

Comment:
You can change this, to remove the unnecessary brackets and adhere to the coding standards 👍 
```php
$webhook['url'] .= ( isset(parse_url($webhook['url'])['query'])) ? '&' : '?' ;
```

Then I'll merge this, thanks!